kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/model/VirtualCluster.java

```diff
@@ -60,6 +65,24 @@ public VirtualCluster(String clusterName,
         // TODO: https://github.com/kroxylicious/kroxylicious/issues/104 be prepared to reload the SslContext at runtime.
         this.upstreamSslContext = buildUpstreamSslContext();
         this.downstreamSslContext = buildDownstreamSslContext();
+        logVirtualClusterSummary(clusterName, targetCluster, clusterNetworkAddressConfigProvider, tls);
+    }
+
+    private static void logVirtualClusterSummary(String clusterName, TargetCluster targetCluster,
+                                                 ClusterNetworkAddressConfigProvider clusterNetworkAddressConfigProvider,
+                                                 Optional<Tls> tls) {
+        try {
+            var downstreamTls = tls.map(tls1 -> " (TLS)").orElse("");
+            HostPort downstreamBootstrap = clusterNetworkAddressConfigProvider.getClusterBootstrapAddress();
+            var upstreamTls = targetCluster.tls().map(tls1 -> " (TLS)").orElse("");
+            HostPort upstreamHostPort = targetCluster.bootstrapServersList().get(0);
+            LOGGER.info("Virtual Cluster: {}, Downstream {}:{}{} => Upstream {}:{}{}",
+                    clusterName, downstreamBootstrap.host(), downstreamBootstrap.port(), downstreamTls,
```

Comment:
nit: does `HostPort` not do a reasonable job of rendering itself that we need to pull it apart manually here?

Reply:
hmm, does exactly what we want :+1: 

toString() {
    return host + ":" + port;
}